src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+
+    public static Ball getRandomBall() {
```

Comment:
The method `getRandomBall()` is currently static. According to the checklist, 'Don't use static methods in your solution.' Consider creating an instance of the `Lottery` class and calling the method on that instance instead.